internal/scan-manager/webapp/src/main/webapp/WEB-INF/views/scan-manager/index.jsp

```diff
@@ -35,10 +35,10 @@
     <hr>
     <div class="row">
         <div class="col-md-12" style="margin-top: 50px; text-align: center; margin-bottom: 20px;">
-            <a class="btn btn-primary scan-manager-index-page-button" href="scanners"
+            <a class="btn btn-primary scan-manager-index-page-button" href="/scan-manager/scanners"
```

Comment:
Is this an intended change? IMU we removed this because we were going to handle the context at Nginx level.

Reply:
@NShani  shall we check this and   merge the PR as early as possible